.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,124 @@
+name: 'Combine PRs'
+
+# This workflow will merge all dependabot PRs into one branch and open a new PR against dependabot-upgrades
```

Comment:
It should open a new PR against dev. 
This will help us also run the static site generation against it to make sure that the changes to not affect rendering.

Reply:
It has to be run manually after all dependabot PRs have been opened.
If we run it automatically for each PR opened by dependabot, only the first dependabot update opens a PR to the dependabot-updates branch. The rest of the updates are rejected as a PR with the same name has already been opened by the first run of the worflow.